akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/SyncTestingExampleSpec.scala

```diff
@@ -120,5 +142,27 @@ class SyncTestingExampleSpec extends AnyWordSpec with Matchers {
       testKit.logEntries() shouldBe Seq(CapturedLogEvent(Level.INFO, "Saying hello to Inboxer"))
       //#test-check-logging
     }
+
+    "has access to the provided config" in {
```

Comment:
@patriknw, I think this provides the test coverage u mentioned in your review summary